promptsource/templates/kilt_tasks/nq/templates.yaml

```diff
@@ -0,0 +1,142 @@
+dataset: kilt_tasks
+subset: nq
+templates:
+  294fd8f3-c7e0-4b3c-abd3-64527f8f71b1: !Template
+    answer_choices: null
+    answer_choices_key: null
+    id: 294fd8f3-c7e0-4b3c-abd3-64527f8f71b1
+    jinja: 'The goal is to predict an English answer string for an input English question.
+      All questions can be answered using the contents of English Wikipedia.
+
+      Question: {{input}}
+
+      Answer:
+
+      |||
+
```

Comment:
@VictorSanh I'm not sure if this is a streamlit rendering issue or a YAML artifact… Is the model expected to generate a newline here?

Reply:
it doesn't matter much because in practise outputs (and inputs) are stripped for left and right white spaces